api/src/test/java/no/bibsys/web/DatabaseResourceTest.java

```diff
@@ -303,6 +311,51 @@ public void updateEntity_EntityExists_ReturnsUpdatedEntity() throws Exception {
         assertThat(actual, is(equalTo(updatedLabel)));
 
     }
+    
+    @Test
+    public void uploadArrayOfThreeEntities_RegistryExists_RegistryContainsThreeEntities() throws Exception {
+        
+        String registryName = UUID.randomUUID().toString();
+        RegistryDto registryDto = sampleData.sampleRegistryDto(registryName);
+        createRegistry(registryDto);
+        
+        
+        List<EntityDto> sampleEntities = new CopyOnWriteArrayList<EntityDto>();
+        EntityDto sampleEntityDto1 = sampleData.sampleEntityDto();
+        sampleEntityDto1.setId("id1");

```

Comment:
Never keen on magic strings and numbers — I'd maybe even randomise test data to create a generic test of our expectations.

Reply:
Randomized the identifiers